src/rtc/rtc_s1_single_job.py

```diff
@@ -1821,15 +1823,16 @@ def run_single_job(cfg: RunConfig):
 
     if save_mosaics:
```

Comment:
There is the same if statement right after this block, around line `1953` as below. Can you consider putting those two blocks into one?

```python
# Append metadata to mosaic GeoTIFFs
    if save_mosaics:
        for current_file in mosaic_output_file_list:
            if not current_file.endswith('.tif'):
                continue
            append_metadata_to_geotiff_file(current_file,
                                            mosaic_geotiff_metadata_dict,
                                            mosaic_product_id)

```

Reply:
@seongsujeong , sorry, I'm having a hard time understanding this comment. You mean that this block is duplicated? I couldn't find the duplicated code. Could you please elaborate?

Reply:
Sorry for not being clear. what I meant is that I've found this pattern:

```python
if save_mosaics:  # line 1825
    [do something]

if save_mosaics:  #line 1954
    [do another thing]

```

So I just thought we can do like:

```python
if save_mosaics:
    [do something]
    [do another thing]

```

Please let me know if it makes sense.

Reply:
Ok, just updated it. Thanks